.devcontainer/Dockerfile.in

```diff
@@ -1,23 +1,17 @@
 FROM %%ENVOY_BUILD_IMAGE%%
 
-ARG USERNAME=vscode
+ARG USERNAME=envoybuild
 ARG USER_UID=501
 ARG USER_GID=$USER_UID
 
 ENV BUILD_DIR=/build
 
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get -y update \
-  && apt-get -y install --no-install-recommends libpython3.13 net-tools psmisc vim 2>&1 \
-  # Create a non-root user to use if preferred - see https://aka.ms/vscode-remote/containers/non-root-user.
-  && groupadd --gid $USER_GID $USERNAME \
-  && groupadd -f pcap \
-  && useradd -s /bin/bash --uid $USER_UID --gid $USER_GID -m $USERNAME -G pcap -d /build \
+  && apt-get -y install --no-install-recommends python3 clang-format net-tools iputils-ping procps psmisc vim 2>&1 \
```

Comment:
May also install a clangd? so the developer need to install it manually, it's inevitable anyway.

Reply:
Done!